contracts/CapabilityDelegator.cdc

```diff
@@ -7,56 +7,56 @@
 /// private `Delegator` can only be borrowed from the child account when you have access to the full `ChildAccount` 
 /// resource.
 ///
-pub contract CapabilityDelegator {
+access(all) contract CapabilityDelegator {
 
     /* --- Canonical Paths --- */
     //
-    pub let StoragePath: StoragePath
-    pub let PrivatePath: PrivatePath
-    pub let PublicPath: PublicPath
+    access(all) let StoragePath: StoragePath
+    access(all) let PrivatePath: PrivatePath
+    access(all) let PublicPath: PublicPath
     
     /* --- Events --- */
     //
-    pub event DelegatorCreated(id: UInt64)
-    pub event DelegatorUpdated(id: UInt64, capabilityType: Type, isPublic: Bool, active: Bool)
+    access(all) event DelegatorCreated(id: UInt64)
+    access(all) event DelegatorUpdated(id: UInt64, capabilityType: Type, isPublic: Bool, active: Bool)
 
     /// Private interface for Capability retrieval
     ///
-    pub resource interface GetterPrivate {
-        pub fun getPrivateCapability(_ type: Type): Capability? {
+    access(all) resource interface GetterPrivate {
+        access(Capabilities) view fun getPrivateCapability(_ type: Type): Capability? {
```

Comment:
Probably should be changed. Entitlements are the main piece in this PR that I feel needs group discussion. Because of that, I went ahead and submitted what was already done so we have _something_ to go off of